<?xml version="1.0"?>
<launch>
  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <param name="/camera/realsense2_camera/filters" value="pointcloud"/>
    <arg name="filters"  value="pointcloud"/>
    <arg name="color_width" value="640"/>
    <arg name="color_height" value="480"/>
    <arg name="color_fps" value="30"/>
  </include>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find apu_medication_support)/rviz/sample.rviz" required="true" output="screen" />

  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="image_rect" to="/camera/depth/image_rect_raw"/>
    <remap from="camera_info" to="/depth_registered/camera_info"/>
  </node>

  <!--
  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/register nodelet_manager">
    <remap from="rgb/camera_info" to="/camera/color/camera_info"/>
    <remap from="depth/camera_info" to="/camera/depth/camera_info"/>
    <remap from="depth/image_rect" to="/camera/depth/image_rect_raw"/>
  </node>

  <node pkg="nodelet" type="nodelet" name="nodelet2"
        args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
    <remap from="rgb/image_rect_color" to="/camera/color/image_raw"/>
    <remap from="rgb/camera_info" to="/depth_registered/camera_info"/>
  </node>
-->
</launch>
